dolphinscheduler-python/pydolphinscheduler/tox.ini

```
@@ -53,6 +53,13 @@ extras = test
 commands =
     python -m pytest tests/integration/
 
+[testenv:local-integrate-test]
+extras = test
+setenv =
+    skip_launch_docker = true
+commands =
+    python -m pytest tests/integration/
```

we can use the macro to use the existing command in L54. we already have the similar usage in L66 - L68
```suggestion
    {[testenv:integrate-test]commands}
```